libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -167,6 +167,26 @@ class UdpContext
 
 #endif // !LWIP_IPV6
 
+    /*
+     * Add a netif (by its index) as the multicast interface
+     */
+    void setMulticastInterface(netif* p_pNetIf)
+    {
+#if LWIP_VERSION_MAJOR == 1
+        udp_set_multicast_netif_addr(_pcb, (p_pNetIf ? ip_2_ip4(p_pNetIf->ip_addr) : ip_addr_any));
```

Comment:
FYI `ip_2_ip4(x)` is like `(x)` [with lwIP-v1](https://github.com/esp8266/Arduino/blob/a2141803f11d91a5654c7a7065b552302cac527c/cores/esp8266/IPAddress.h#L33) (because IPv6 does not exist in it)

Reply:
Updated the PR